README.md

```diff
@@ -149,22 +153,28 @@ After acquiring the token, you can put it in Authorize section on the http://loc
 
 `/tasks/schedule` endpoint accepts the following payload:
 
-```
+```ruby
 {
-  "runner_type": "string", # the instance backend to use (docker, opennebula, etc.). For now only `docker` and `any` values are supported
-  "dist_name": "string", # name of distribution you want to test on (debian, ubuntu, centos, etc.)
-  "dist_version": "string", # distribution version (20.04, 8, etc.)
-  "dist_arch": "string", # CPU architecture you want to test package for. Supported values - 'x86_64', 'i686', 'amd64', 'aarch64', 'arm64'
-  "repositories": [], # a list of repositories to add before attempting package installation. Each repository is a dictionary with `name` and `url` values. name` is optional
-  "package_name": "string", # the name of the package to install
+  "runner_type": "string" # the instance backend to use (docker, opennebula, etc.). For now only 'docker' and 'any'' values are supported
+  "dist_name": "string" # name of distribution you want to test on (debian, ubuntu, centos, etc.)
+  "dist_version": "string" # distribution version (20.04, 8, etc.)
+  "dist_arch": "string" # CPU architecture you want to test package for. Supported values - 'x86_64', 'i686', 'amd64', 'aarch64', 'arm64'
+  "repositories": [] # a list of repositories to add before attempting package installation. Each repository is a dictionary with 'name' and 'url' values. 'name' is optional
+  "package_name": "string" # the name of the package to install
   "package_version": "string" # optional, version of the package you want to install
 }
 ```
   
 `/tasks/{task_id}/result` endpoint returns the result of task. `task_id` - task ID string.
 
+
 Unresolved issues
 --
 - Provision for production server is required;
```

Comment:
We have provision and very basic test suite, I'd add something like "No support for external test projects like LTP"

Reply:
Done